src/main/webapp/college-info.js

```diff
@@ -293,4 +307,25 @@ function prepReviewForm(id) {
   idInputElement.setAttribute('value', id);
 }
 
+/**
+ * Sets the hidden elements to be shown and the shown elements hidden.
+ * ID for buttons that will be used to show an element must be the same as the
+ * name of that element ID with "-nav" at the end.
+ * @param {string} showId HTML ID of element that is hidden and will be shown.
+ * @param {string} hideId HTML ID of element that is shown and will be hidden.
+ */
+function toggleElementsDisplay(showId, hideId) {
+  const elemnetToHide = document.getElementById(hideId);
+  const elementToShow = document.getElementById(showId);
+
+  elemnetToHide.style.display = 'none';
```

Comment:
I'd suggest simplifying this by adding and removing an `is-hidden` class in CSS. Or, another option could be to add a class to the `main` element like `active-section-info` and `active-section-reviews` and in CSS those would take care of showing/hiding the right section and giving the correct button the active state.